src/openforms/submissions/tests/test_submission_attachment.py

```diff
@@ -1060,9 +1087,13 @@ def test_attach_upload_larger_than_configured_max_size_raises_413(self):
                 "fileMaxSize": "10B",
             },
         ]
+
+        # Using `legacy` is the only way to create an upload without a submission.
+        # It will be set later.
         upload = TemporaryFileUploadFactory.create(
-            file_name="aaa.txt", content__data=b"a" * 20, file_size=20
+            file_name="aaa.txt", content__data=b"a" * 20, file_size=20, legacy=True
         )
```

Comment:
why can't we create it with a submission from the get-go?

Reply:
Because the submission is created afterwards with `.from_components(components, data)`, and `data` is created using `upload`

Reply:
you can just call the submission creation before without any data, and then set the submission step data, no? That also better reflects what actually happens with the SDK - the submission must exist before uploads/step data submission can happen.